### PR TITLE
{vis}[GCCcore/8.3.0] Sphinx v2.4.3 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by furstj** [comment]
@boegel Finally, I had to add some additional modules to overcome difficulties with sanity_pip_check.

File: easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+
+exts_list = [
+    ('packaging', '20.1', {
+        'checksums': ['e665345f9eef0c621aa0bf2f8d78cf6d21904eef16a93f020240b704a57f1334'],
+    }),
+    ('MarkupSafe', '1.1.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb]: @furstj This extension is already include in Python 3.7.4 (exact same version).

`packaging` is there already too, but a slightly older version (19.1), is the newer version strictly necessary?

Comment (furstj) [reply easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb]: @boegel, it seems that the 2.2.0 version of Sphinx is sufficient for me. So I will close this pull request. I'm sorry for wasting your time and thank you for helpful advices that I will certainly use for next PRs.

Comment (boegel) [reply easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb]: Absolutely no problem, I understand it's sometimes easy to overlook how things change between easyconfig generations...

Thanks a lot for all your efforts!

**Pull request closed without merging**